app/views/hyku_addons/account_settings/index.html.erb

```diff
@@ -19,36 +19,34 @@
 
     <% ['contact_email', 'gtm_id', 'demo_gtm_id', 'live', 'institutional_relationship', 'creator_fields', 'contributor_fields'].each do |fname| %>
       <tr>
-        <td id= "link-<%= fname %>"><b><%= fname %>:</b> <%= @account && @account.settings && @account.settings[fname] %> </td> <br>
-        <td><%= link_to "Edit #{fname}", hyku_addons.edit_admin_account_setting_path(id: @account, partial_name: 'render_text_fields', field_name: fname), class: "btn btn-xs btn-default",  remote: true %></td>
+        <td id= "link-<%= fname %>"><b><%= fname.humanize %>:</b> <%= @account && @account.settings && @account.settings[fname] %> </td> <br>
```

Comment:
I think we can extract this into a helper method + presenters.
Something like:

<td><%= link_to_edit_account_settings %></td>

https://www.rubyguides.com/2019/09/rails-patterns-presenter-service/